config/routes.js

```diff
@@ -146,6 +146,7 @@ module.exports.routes = {
   'GET    /noo/logout':                                   'SessionController.destroy',
 
   'POST    /noo/pn/add-device':                           'PushNotificationController.addDevice',
+  'POST    /noo/pn/disable-device':                       'PushNotificationController.disableDevice',  
```

Comment:
cool, so would it work if it were 
`POST /noo/device/:token => DeviceController.create` 
and 
`DELETE /noo/device/:token => DeviceController.destroy`
as the create method needs the token as well?